### DISCUSSION TITLE
Titanium Implants and Ultralight Frames rebalance

### DISCUSSION THREAD

**Discussion by MartenKahr** [cataclysmbn/Cataclysm-BN#3240] [General]
### Is your feature request related to a problem? Please describe.

It seems extremely strange to me that Titanium Implants, with a volume of 0.500L, that can regularly be dissected from zombie corpses, produce 5 scrap titanium (0.100L each), and that amount of scrap titanium is enough to craft 2.5 Ultralight frames with a volume of 7.50L each. Although the mass of the items involved checks out, the volume aspect certainly does not. Another strangeness here is the inexplicable abundance of titanium implants: practically every zombie seems to have a 2.25kg titanium implant in them that can potentially be recovered. Being able to put together the entire chassis of something like a light buggy from the hip replacements and bone reinforcements of no more than ten people just seems off to me, especially when practically every zombified person seems to have one in them.

### Describe the solution you'd like

Reduce the mass of the titanium implants to 0.500kg, and have the item produce 5 titanium instead of 5 scrap titanium.

### Describe alternatives you've considered

Another possibility is drastically reducing the abundance of titanium implants.

**Comment by RelMayers** [comment]
Aren't frames supposed to have a lot of empty space?

**Comment by MartenKahr** [comment]
They are, but being able to get enough titanium to build two bicycle frames (that admittedly need additional parts to be running bikes) out of three zombie corpses, "ultra-light frame" or not, sounds a bit far-fetched. Given that titanium has been made the "ultralight" material in the game, lighter than aluminium, I think the current mass of the implants is unrealistically high, and that's producing less-than-sane results downstream.

**Comment by RelMayers** [comment]
In my opinion such implants probably would yield enough resources to justify this.
Strength required to make frame is dependent on material strength, good material means less mass required for same durability.

**Comment by MartenKahr** [comment]
In my opinion, if it's a good material that needs less mass for the same durability, why is almost an entire bicycle frame's worth of that material necessary for minor bone reinforcements that practically every person pre-apocalypse had installed in them?

**Comment by RelMayers** [comment]
"Road bikes will weigh between 6.72kg and 9.57kg on average." 
And this is probably whole bike not just frame.
